common/views/components/Body/Body.js

```diff
@@ -165,6 +168,15 @@ const Body = ({ body, isDropCapped }: Props) => {
               </Layout10>
             )}
 
+            {slice.type === 'collectionVenue' && (
+              <VenueHours
+                venue={parseVenueTimesToOpeningHours(slice.value)}
+                isInList={
+                  pageId === 'openingTimes' || pageId === 'WwQHTSAAANBfDYXU'
```

Comment:
What's going on here?

Reply:
The `VenueHours` component behaves slightly differently if it's on the opening-times page, or on a venue page.

Although we have a vanity url for `opening-times`, the opening times page content comes from Prismic and it available at `/pages/WwQHTSAAANBfDYXU`, so makes sense to check for either.